src/NexusMods.App.UI/Pages/Sorting/LoadOrder/LoadOrderViewModel.cs

```diff
@@ -117,13 +117,72 @@ public LoadOrderViewModel(LoadoutId loadoutId, ISortableItemProviderFactory item
                                 }
                             );
                             
-                            if (!item.HasValue)
-                            {
-                                return;
-                            }
+                            if (!item.HasValue) return;
                             await provider.SetRelativePosition(item.Value, delta, cancellationToken);
+                        })
+                    .DisposeWith(d);
+
+                // Drag and drop
+                adapter.RowDragOverSubject
+                    .Subscribe(dragDropPayload =>
+                        {
+                            var (sourceModels, targetModel, eventArgs) = dragDropPayload;
+
+                            if (eventArgs.Position != TreeDataGridRowDropPosition.Inside) return;
+                            
+                            // Update the drop position for the inside case to be before or after
+                            var positionY = eventArgs.Inner.GetPosition(eventArgs.TargetRow).Y / eventArgs.TargetRow.Bounds.Height;
+                            eventArgs.Position = positionY < 0.5 ? TreeDataGridRowDropPosition.Before : TreeDataGridRowDropPosition.After;
```

Comment:
Why are we changing the position when TreeDataGrid returns `TreeDataGridRowDropPosition.Inside`?

Reply:
For UI purposes. The drop indicator line will show either Before or after rather than showing inside.

Since we don't support dropping inside for the moment, this is the approach that offers the best experience to users.
Otherwise trying to hit the Before or After becomes finnicky and annoying.

Reply:
Since this code is repeated 3 times in this file, you should make that a method.

Reply:
Made a method but I'm not sure how much clearer it is. 
If you have a better name for the method let me know.